src/components/views/rooms/SendMessageComposer.tsx

```diff
@@ -277,7 +277,7 @@ export class SendMessageComposer extends React.Component<ISendMessageComposerPro
     private currentlyComposedEditorState: SerializedPart[] | null = null;
     private dispatcherRef: string;
     private sendHistoryManager: SendHistoryManager;
-    private imagePack: object;
+    private imagePack: Record<string,Record<string,Record<string,string>>>;;
```

Comment:
Can you use an interface here and give these things names? its impossible to understand this type from this definition. `{[key: string]: {...}}` etc

Reply:
Added interface with clearer definitions